Library/Extensions/UIView+Helper.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+extension UIView {
+  public func rounded(with cornerRadius: CGFloat = Styles.cornerRadius) {
+    self.clipsToBounds = true
+    self.layer.masksToBounds = true
+    self.layer.cornerRadius = cornerRadius
+  }
```

Comment:
Hmm do we not have a helper for this somewhere already? I might be thinking of a different one 🤔

Reply:
Yes, we do have one in Prelude version as `Styles/BaseStyles.roundedStyle`. This new implementation is a `UIView` extension, designed to provide a more reusable and flexible approach directly within the UIKit layer.